hazelcast/src/test/java/com/hazelcast/nio/tcp/BindHandlerTest.java

```diff
@@ -208,57 +226,25 @@ private Packet extendedBind() {
         return addresses;
     }
 
-    private static Map<EndpointQualifier, Address> serverAddresses() {
-        Map<EndpointQualifier, Address> addressMap = new HashMap<EndpointQualifier, Address>();
-        addressMap.put(EndpointQualifier.MEMBER, SERVER_MEMBER_ADDRESS);
-        addressMap.put(EndpointQualifier.CLIENT, SERVER_CLIENT_ADDRESS);
-        addressMap.put(EndpointQualifier.resolve(WAN, "wan"), SERVER_WAN_ADDRESS);
-        return addressMap;
-    }
-
-    private void setup(final ConcurrentHashMap<Address, TcpIpConnection> connectionsMap,
-                       ProtocolType protocolType) {
-        ILogger logger = Logger.getLogger(BindHandlerTest.class);
-
-        LoggingService loggingService = mock(LoggingService.class);
-        when(loggingService.getLogger(any(Class.class))).thenReturn(logger);
-
-        IOService ioService = mock(IOService.class);
-        when(ioService.getSerializationService()).thenReturn(serializationService);
-        when(ioService.getLoggingService()).thenReturn(loggingService);
-        when(ioService.getThisAddress()).thenReturn(SERVER_MEMBER_ADDRESS);
-        when(ioService.getThisAddresses()).thenReturn(serverAddresses());
-
-        NetworkingService networkingService = mock(NetworkingService.class);
-        when(networkingService.getIoService()).thenReturn(ioService);
-
-        endpointManager = mock(TcpIpEndpointManager.class);
-        when(endpointManager.getNetworkingService()).thenReturn(networkingService);
-        when(endpointManager.registerConnection(any(Address.class), any(TcpIpConnection.class)))
-                .then(new Answer<Boolean>() {
-                    @Override
-                    public Boolean answer(InvocationOnMock invocation)
-                            throws Throwable {
-                        connectionsMap.put(
-                                (Address) invocation.getArgument(0),
-                                (TcpIpConnection) invocation.getArgument(1)
-                        );
-                        return true;
-                    }
-                });
-        when(endpointManager.getEndpointQualifier()).thenReturn(EndpointQualifier.resolve(protocolType, randomName()));
-        setInternalState(endpointManager, "connectionsInProgress", new HashSet<Address>());
-        setInternalState(endpointManager, "connectionsMap", connectionsMap);
-
-        Socket socket = mock(Socket.class);
-        when(socket.getRemoteSocketAddress()).thenReturn(CLIENT_SOCKET_ADDRESS);
-
-        channel = mock(Channel.class);
-        ConcurrentMap channelAttributeMap = new ConcurrentHashMap();
-        when(channel.attributeMap()).thenReturn(channelAttributeMap);
-        when(channel.socket()).thenReturn(socket);
-        when(channel.remoteSocketAddress()).thenReturn(CLIENT_SOCKET_ADDRESS);
-
-        bindHandler = new BindHandler(endpointManager, ioService, logger, true, Collections.singleton(protocolType));
+    private Config createConfig() {
```

Comment:
The configuration returned has multicast join enabled with the default config. From the test and environment stability perspective, it seems safer to use TCP joins.

Reply:
`TestAwareInstanceFactory` takes care of that one as well, thanks!